### ISSUE TITLE
How to add Menu to Top Right of Appbar?

### ISSUE THREAD

**Issue by ghprod** [museui/muse-ui#71]
Hi, 

is this bug? i try to put menu on top right of appbar, but its break.

menu item width was clipped and has scroll inside it.

![image](https://cloud.githubusercontent.com/assets/1922652/20478852/809d8e70-b00e-11e6-80e7-b72a514858b3.png)

![image](https://cloud.githubusercontent.com/assets/1922652/20478847/7bff3616-b00e-11e6-8a11-4a61bd703cc7.png)

this is my code on jade syntax

```jade
    mu-appbar.example-appbar(:zDepth='0', :title='title', :class="{'nav-hide': !open}")
      mu-icon-button(@click='toggleNav', icon='menu', slot='left')
      mu-icon-menu(
        slot="right"
        icon="more_vert"
      )
        mu-menu-item(title="Sign out")
```

Thanks

**Comment by ghprod** [comment]
If i add two menu item then everything is ok

![image](https://cloud.githubusercontent.com/assets/1922652/20479804/320e5088-b012-11e6-81be-12882db95ab6.png)

Its break only when has one menu item

Thanks

**Comment by ghprod** [comment]
Update:

Seems like container need to apply `white-space: nowrap` or something like that, because its break when first menu not long enough or contain two word

![image](https://cloud.githubusercontent.com/assets/1922652/20479968/0303440a-b013-11e6-87e8-15d5d90f034c.png)

**Comment by myronliu347** [comment]
Yes, this seems to be a problem，thank you!

**Comment by myronliu347** [comment]
fixed in [2.0.0-alpha.9](https://museui.github.io/)
